src/BenchmarksApps/TechEmpower/PlatformBenchmarks/BenchmarkApplication.Fortunes.cs

```diff
@@ -46,12 +48,16 @@ private void OutputFortunes(PipeWriter pipeWriter, List<Fortune> model)
             var bodyStart = writer.Buffered;
             // Body
             writer.Write(_fortunesTableStart);
+            Span<byte> htmlEncodedMessage = stackalloc byte[256];
```

Comment:
Could avoid the `stackalloc` completely and get a buffer from the `writer` to encode into manually followed by calling `Advance`